apps/graphql/src/dataloaders/Itinerariesloader.js

```diff
@@ -34,11 +35,74 @@ export type RouteItem = {|
   +id: string,
 |};
 
+// start - new structure
+
+export type Country = {|
+  +id: string,
+  +name: string,
+  +code: string,
+  +slug: string,
+  +flagURL: string,
+|};
+
+// @TODO rename PriceType
 export type Price = {|
   +amount: number,
   +currency: string,
 |};
 
+export type LocationType = {|
+  +id: string,
+  +locationId: string,
+  +name: string,
+  +timezone: string,
+  +country: Country,
+|};
+export type DateType = {|
+  +local: Date,
+  +utc: Date,
+|};
+export type ProviderType = {|
+  +id: string,
+  +name: string,
+|};
+export type VehicleType = {|
+  +type: string,
```

Comment:
this can be `enum` or it is not a good idea? (// bus | train | aircraft,)

Reply:
Done